src/App.tsx

```diff
@@ -26,70 +26,104 @@ type ReorderOptions = {
   isReversed: boolean,
 };
 
-// Use this function in the render method to prepare goods
 export function getReorderedGoods(
   goods: string[],
   { sortType, isReversed }: ReorderOptions,
 ) {
-  // To avoid the original array mutation
   const visibleGoods = [...goods];
 
-  // Sort and reverse goods if needed
-  // eslint-disable-next-line no-console
-  console.log(sortType, isReversed);
+  if (sortType === 1) {
```

Comment:
don't use enum values, especially if it's a numbers
do it in this way:
```suggestion
  if (sortType === SortType.LENGTH) {
```
fix everywhere